app/scripts/views/components/virtualStudy/virtualStudy.js

```diff
@@ -102,9 +106,38 @@
       enableShareCohortBtn: function(tooltip) {
         this.disableBtn(tooltip.find('.share-cohort'), false);
       },
-      getDefaultVSDescription: function() {
-        var def = new $.Deferred();
-        var selectedStudiesDisplayName = window.iviz.datamanager.getCancerStudyDisplayName(_.pluck(this.stats.studies, 'id'));
+      getFilteredOriginStudies: function() {
+        var selectedStudiesDisplayName = window.iviz.datamanager.getCancerStudyDisplayName(this.stats.origin);
```

Comment:
does the `getCancerStudyDisplayName` function handles VS?

Reply:
It does.

Reply:
I think it just handles saved VS but not shared VS